src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -241,6 +261,8 @@ private BooleanSupplier isClimberAtPosition(Angle position) {
   }
 
   public Command climb() {
+    currentLimit.SupplyCurrentLimit = 30;
+    currentLimit.StatorCurrentLimit = 30;
```

Comment:
This doesn't apply the currentLimit. Need to call apply() like in the constructor. 
Also, it needs to be part of the Command